CHANGES.rst

```diff
@@ -1,3 +1,10 @@
+22.3.0
+======
+
+- Fix ``PerFactException`` handling for charmeleon-based page templates.
```

Comment:
typo: chameleon, not charmeleon.

Reply:
Woops, confused it probably with writing of Charmander...fix it right away!